### PR TITLE
Enhancement/skale 4212 filestorage https

### PR DESCRIPTION
**Pull request by DmytroNazarenko**
* Regenerate nginx config after ssl upload

### CODE REVIEW

File: core/schains/ssl.py [hunk 1]
```diff
@@ -25,6 +25,12 @@ def is_ssl_folder_empty():
     return len(os.listdir(SSL_CERTIFICATES_FILEPATH)) == 0
 
 
+def check_ssl_certs():
```
Comment (badrogger) [inline core/schains/ssl.py]: `is_ssl_folder_empty` serves the same purpose

File: web/routes/ssl.py [hunk 2]
```diff
@@ -87,6 +88,7 @@ def upload():
         ssl_cert = request.files[SSL_CRT_NAME]
         ssl_key.save(os.path.join(SSL_CERTIFICATES_FILEPATH, SSL_KEY_NAME))
         ssl_cert.save(os.path.join(SSL_CERTIFICATES_FILEPATH, SSL_CRT_NAME))
+        generate_nginx_config()
```
Comment (badrogger) [inline web/routes/ssl.py]: It's better to add error handling here. If nginx reloading failed we still can reload schains.

File: core/nginx.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+def generate_nginx_config():
+    ssl_on = check_ssl_certs()
+    template_data = {
+        'ssl': ssl_on,
+    }
+    logger.info(f'Processing nginx template. ssl: {ssl_on}')
+    process_template(NGINX_TEMPLATE_FILEPATH, NGINX_CONFIG_FILEPATH, template_data)
+    nginx_container = dutils.client.containers.get(NGINX_CONTAINER_NAME)
+    nginx_container.restart()
```
Comment (badrogger) [inline core/nginx.py:28]: I suggest to move restarting to separate function in order to test them separately.

**Review by badrogger — CHANGES_REQUESTED** [review]

File: core/schains/ssl.py (hunk 1 above)
Comment (DmytroNazarenko) [reply core/schains/ssl.py]: We need to check both key and cert

**Review by badrogger — APPROVED** [review]

**Review by dmytrotkk — APPROVED** [review]

**Pull request merged**